cmd/maya-apiserver/cstor-operator/spc/handler.go

```diff
@@ -176,7 +177,7 @@ func validatePoolType(spc *apis.StoragePoolClaim) error {
 func validateDiskType(spc *apis.StoragePoolClaim) error {
 	diskType := spc.Spec.Type
 	// TODO: Update below snippet to make use of predicates
-	if !(diskType == string(apis.TypeBlockDeviceCPV)) {
+	if !spcv1alpha1.SupportedDiskType[diskType] {
```

Comment:
This is not relevant to this commit.
How does the error message knows that this function is being used for `storagepool create operation`

Reply:
Before creating the pools we are validating the `spc.yaml` provided by the user, caller of this function will have leading error message `Validation of spc failed:`